src/Sql/dbo/Stored Procedures/Cipher_DeleteAttachment.sql

```diff
@@ -11,6 +11,7 @@ BEGIN
     DECLARE @UserId UNIQUEIDENTIFIER
     DECLARE @OrganizationId UNIQUEIDENTIFIER
 
+    // TODO: Is this select statement doing anything?
```

Comment:
Yes, it is grabbing the user and organization ids back out of the Cipher table for the given Cipher record ID, and then using that to go update the appropriate storage and revision date values downstream.